01-numpy.md: change weight to avoid numerical errors

joshkyh opened the pull request:
Proposed fix to floating point print by through native function format #328 

---
_edited by @maxim-belkin to remove PR template_

MikeTrizna commented:
I agree with @maxim-belkin that it would be simpler to change `weight` from 55 to 60 (or any other integer that doesn't produce this behavior). 

I've personally run into this behavior early in a Python workshop, and having to explain what was happening caused a major delay AND left a few participants concerned that they would be better off continuing to do calculations in Excel...

maxim-belkin commented:
Looks like @tbekolay suggested the same solution a while back here: https://github.com/swcarpentry/python-novice-inflammation/pull/339#issuecomment-339103601

@joshkyh can you update your pull request to change the weight from 55 to 60?

maxim-belkin commented:
Also, is this part of instructor checkout?

joshkyh commented:
@maxim-belkin done - pull request approach changed to simply changing the weight to 60 :) Not sure about your question on "is this part of instructor checkout". Is this question meant for me?

joshkyh commented:
Hi @maxim-belkin Apologies, I should have looked through the rest of the file in detail. I have changed the rest of the code that referenced the original weight and changed it accordingly.

I did some search with reference to "instructor checkout" procedure, I am not part of any instructor training programme and does not require this commit to be logged against my progress - just a member of the public. :)

Thanks,
Joshua

tbekolay commented on _episodes/01-numpy.md:
```diff
@@ -53,19 +53,21 @@ print(weight_kg)
 
 ~~~
-weight in pounds: 121.0
+weight in pounds: 120.0
```
I believe this should be `132.0.`

joshkyh commented:
@tbekolay yes you're right, my bad. Thanks for pointing out.
